Assets/Sequence/SequenceSDK/WaaS/WaaSLogin.cs

```diff
@@ -31,8 +32,85 @@ public class WaaSLogin : ILogin
         private string _sessionId;
 
         private bool _isLoggingIn = false;
+        
+        private bool _storeSessionWallet = false;
+        private const string _walletKey = "SessionWallet";
+        private const string _waasWalletAddressKey = "WaaSWalletAddress";
 
         public WaaSLogin(IValidator validator = null)
+        {
+            bool storeSessionWallet = SequenceConfig.GetConfig().StoreSessionPrivateKeyInSecureStorage;
+            if (storeSessionWallet)
+            {
+                _storeSessionWallet = true;
+                Configure();
+                TryToLoginWithStoredSessionWallet();
+            }
+            else
+            {
+                CreateWallet(validator);
+            }
+        }
+
+        private void CreateWallet(IValidator validator = null)
+        {
+            Configure();
+
+            SetupAuthenticator(validator);
+        }
+
+        public void SetupAuthenticator(IValidator validator = null)
+        {
+            ConfigJwt configJwt = SequenceConfig.GetConfigJwt();
+            _sessionWallet = new EthWallet();
+            _sessionId = IntentDataOpenSession.CreateSessionId(_sessionWallet.GetAddress());
+
+            string nonce = SequenceCoder.KeccakHashASCII(_sessionId).EnsureHexPrefix();
+            if (OpenIdAuthenticator.Instance == null)
+            {
+                SetAuthenticatorAndSetupListeners(nonce);
+            }
+            else
+            {
+                SetAuthenticator(nonce);
+            }
+
+            if (validator == null)
+            {
+                validator = new Validator();
+            }
+            _validator = validator;
+
+            try
+            {
+                _emailSignIn = new AWSEmailSignIn(configJwt.emailRegion, configJwt.emailClientId, nonce);
+            }
+            catch (Exception e)
+            {
+                Debug.LogWarning("AWS config not found in config key. Email sign in will not work. Please contact Sequence support for more information.");
```

Comment:
Most integrators won't have email auth. I think this might be a bit misleading.

Reply:
Do we have a doc I can link to on how to setup email auth in the builder? Maybe I could change the warning to:
"AWS config not found in config key. Email sign in will not work. Please see this doc to learn how to configure your project for email sign in: "

What do you think?

Reply:
For now it's better to silently fail. Because our current email auth is not open to all partners. When we have it fully live, we can add an info log with a link to docs.